cmd/deployctl/cmd/build.go

```diff
@@ -167,14 +168,15 @@ func CollectCmdRun(cmd *cobra.Command, args []string) {
 	}
 
 	if minimalConfig {
-		noCACertificates = true
 		noDefaults = true
 		noInterfaceDefaults = true
 		normalizeInterfaces = true
 		normalizeMTU = true
 		normalizeConsole = true
 		noServiceParams = true
 	}
+	noDRBDLinkUtilization = true
```

Comment:
Please add a description above to indicate the purpose of set these values

Reply:
I have removed the variable above